relay-base-schema/src/data_category.rs

```diff
@@ -18,133 +18,199 @@ pub struct UnknownDataCategory(pub u8);
 #[repr(i8)]
 pub enum DataCategory {
     /// Reserved and unused.
+    ///
+    /// SDK rate limiting behavior: ignore.
     Default = 0,
     /// Error events and Events with an `event_type` not explicitly listed below.
+    ///
+    /// SDK rate limiting behavior: apply to envelope items of type `event`.
     Error = 1,
     /// Transaction events.
+    ///
+    /// SDK rate limiting behavior: apply to envelope items of type `transaction`.
     Transaction = 2,
     /// Events with an event type of `csp`, `hpkp`, `expectct` and `expectstaple`.
+    ///
+    /// SDK rate limiting behavior: ignore (?).
```

Comment:
See https://develop.sentry.dev/sdk/data-model/envelope-items/#reserved-types



```suggestion
    /// SDK rate limiting behavior: ignore.
```